platform/src/apis/EducationBenchmarking.Platform.Api.Establishment/Db/FieldNames.cs

```diff
@@ -0,0 +1,53 @@
+namespace EducationBenchmarking.Platform.Api.Establishment.Db;
+
+public static class FieldNames
+{
+        public const string URN = "URN";
+        public const string ESTAB_NAME = "EstablishmentName";
+        public const string ESTAB_NAME_UPPERCASE = "EstablishmentNameUpperCase";
+        public const string OVERALL_PHASE = "OverallPhase";
+        public const string PHASE_OF_EDUCATION = "PhaseOfEducation";
+        public const string TYPE_OF_ESTAB = "TypeOfEstablishment";
+        public const string STREET = "Street";
+        public const string TOWN = "Town";
+        public const string POSTCODE = "Postcode";
+        public const string ADDRESS = "FullAddress";
+        public const string LOCATION = "Location";
+        public const string TRUSTS = "Trusts";
+        public const string SPONSORS = "SchoolSponsors";
+        public const string LA_CODE = "LACode";
+        public const string ESTAB_NO = "EstablishmentNumber";
+        public const string LA_ESTAB = "LAEstab";
+        public const string TEL_NO = "TelephoneNum";
+        public const string NO_PUPIL = "NumberOfPupils";
+        public const string STAT_LOW = "StatutoryLowAge";
+        public const string STAT_HIGH = "StatutoryHighAge";
```

Comment:
Is STAT_LOW and STAT_HIGH intuitive enough for someone to know it relates to age?

Reply:
Is this something we can do now? Or not yet since we are still using the SFB datasets?

Reply:
My preference for these field name constants is to keep them as is for now (just because of the serialisation issues and badly named fields). However, everything else most definitely.